src/ctapipe/containers.py

```diff
@@ -1132,40 +1111,172 @@ class TelescopePointingContainer(Container):
+        " for each pixel. Shape: (n_channels, n_pixels)",
+    )
+    outlier_mask = Field(
+        None,
+        "Boolean mask indicating which pixels are considered outliers."
+        " Shape: (n_channels, n_pixels)",
+    )
+    is_valid = Field(
+        False,
+        (
+            "True if the coefficients are valid, False if they are not valid or "
+            "if a high fraction of faulty pixels exceeding the pre-defined threshold "
+            "is detected during the time period."
+        ),
+    )
+
 
+class StatisticsContainer(Container):
+    """Store descriptive statistics of a pixel-wise quantity for each channel"""
 
-class EventCalibrationContainer(Container):
+    mean = Field(
+        None,
+        "mean of a pixel-wise quantity for each channel"
+        "Type: float; Shape: (n_channels, n_pixel)",
+    )
+    median = Field(
+        None,
+        "median of a pixel-wise quantity for each channel"
+        "Type: float; Shape: (n_channels, n_pixel)",
+    )
+    std = Field(
+        None,
+        "standard deviation of a pixel-wise quantity for each channel"
+        "Type: float; Shape: (n_channels, n_pixel)",
+    )
+    n_events = Field(-1, "number of events used for the extraction of the statistics")
+    outlier_mask = Field(
+        None,
+        "Boolean mask indicating which pixels are considered outliers."
+        " Shape: (n_channels, n_pixels)",
+    )
+    is_valid = Field(
+        False,
+        (
+            "True if the pixel statistics are valid, False if they are not valid or "
+            "if a high fraction of faulty pixels exceeding the pre-defined threshold "
+            "is detected across the chunk of images."
+        ),
+    )
+
+
+class ChunkStatisticsContainer(StatisticsContainer):
+    """Store descriptive statistics of a chunk of images"""
+
+    time_start = Field(NAN_TIME, "high resolution start time of the chunk")
+    time_end = Field(NAN_TIME, "high resolution end time of the chunk")
+    event_id_start = Field(None, "event id of the first event of the chunk")
+    event_id_end = Field(None, "event id of the last event of the chunk")
+
+
+class PixelStatisticsContainer(Container):
     """
-    Container for calibration coefficients for the current event
+    Container for pixel statistics from flat-field and sky pedestal events
+    """
+
+    flatfield_image = Field(
+        default_factory=StatisticsContainer,
+        description="Statistical description from the image charge of flat-field event distributions",
+    )
+    flatfield_peak_time = Field(
+        default_factory=StatisticsContainer,
+        description="Statistical description from the peak arrival time of flat-field event distributions",
+    )
+    sky_pedestal_image = Field(
+        default_factory=StatisticsContainer,
+        description="Statistical description from the image charge of sky pedestal event distributions",
+    )
+
+
+class MonitoringCameraContainer(Container):
```

Comment:
Word order reads weird here for me.

Doesn't `CameraMonitoringContainer` and `TelescopeMonitoringContainer` make more sense?

Reply:
Monitoring here stands as a meta-prefix, slightly reflecting the hierarchy. While I agree it reads a bit odd, I'd leave it as is.